systest/scripts/conf/tempest.conf

```diff
@@ -7,7 +7,7 @@ api_version = 2.0
 project_network_cidr = 10.2.4.0/24 
 
 [DEFAULT]
-debug = True
+debug = False
```

Comment:
Yes, I tested it with 2 of the 8 (for time reasons) and it is fine.  This really just helps trim a lot of the output for failed tests which dump all the stdout that they collected during their tests.